rust/migrate-wicked/src/interface.rs

```diff
@@ -22,6 +22,8 @@ pub struct Interface {
     pub ipv6_auto: Option<Ipv6Auto>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub dummy: Option<Dummy>,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Maybe 
```
#[skip_serializing_none]
```
the next time?

Reply:
I just used what was used for the rest of the options. Refactoring isn't the goal of this PR :slightly_smiling_face: